web/pwa/components/ActivityRevision/ActivityRevision.js

```diff
@@ -48,6 +48,7 @@ export default function ActivityRevisionOrCreationModal({
   allowTeamMode = false,
   nullableEndTime = true,
   allowSupportActivity = true,
+  allowTransfers,
```

Comment:
Shouldn't we use a default value?

Reply:
I switched to allowTransfers = false
I don't think I like the idea of needing a default value here though... In theory at least